### PR TITLE
Strip 'px' from numeric values when inlining table,td,th height and width attributes

### PR DESCRIPTION
**Pull request by ryantate13**
Correct format for inlining these values is width="200" vs width="200px". This PR handles case insensitive stripping of 'px' from the end of the attribute value when the key is height or width.

### CODE REVIEW

File: lib/setTableAttrs.js [hunk 1]
```diff
@@ -34,7 +35,14 @@ var cheerio = require('cheerio'),
             styleVal = $el.css(style);
 
             if (styleVal !== undefined) {
-                $el.attr(styleToAttrMap[style], styleVal);
+                $el.attr(
+                    styleToAttrMap[style],
+                    (attributesToRemovePxFrom.indexOf(style) > -1)
```
Comment (jonkemp) [inline lib/setTableAttrs.js]: This is a style issue but I think there is a better way to handle this. Maybe create a variable for it? I don't like including the ternary assignment here or splitting it into multiple lines like that.

File: test/fixtures/table-attr.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
         background-color: red;
         float: left;
         width: 100%;
-        height: 500px;
+        height: 500PX;
```
Comment (jonkemp) [inline test/fixtures/table-attr.html]: Why is this change needed?

File: lib/setTableAttrs.js (hunk 1 above)
Comment (ryantate13) [reply lib/setTableAttrs.js]: I'm not married to it. You prefer if statements to ternaries?

File: test/fixtures/table-attr.html (hunk 2 above)
Comment (ryantate13) [reply test/fixtures/table-attr.html]: The regex to replace px values is case insensitive. This tests that `PX` which is valid CSS will also be removed.

File: lib/setTableAttrs.js (hunk 1 above)
Comment (ryantate13) [reply lib/setTableAttrs.js]: Updated in master :)

**Comment by jonkemp** [comment]
Lint task is failing so the build is failing now. 😄

**Comment by ryantate13** [comment]
> Lint task is failing so the build is failing now. 😄

Fixed 😄
